cherrypy/_cpreqbody.py

```diff
@@ -724,14 +738,22 @@ def default_proc(self):
                 self.file = result
 
     def read_into_file(self, fp_out=None):
-        """Read the request body into fp_out (or make_file() if None).
+        """Read the request body into fp_out.
+
+        if fp_out not given, use either make_file or
+        a BytesIO object, based on maxrambytes.
 
         Return fp_out.
         """
-        if fp_out is None:
-            fp_out = self.make_file()
-        self.read_lines_to_boundary(fp_out=fp_out)
-        return fp_out
+        if not fp_out:
+            x = self.read_lines_to_boundary()
+            if isinstance(x,(str,bytes)):
```

Comment:
Seems to work on both in the interactive shell, bytes appears to be an alias of string in python. There's probably a better way, but this seemed like a decent way to express "Is this value object and not a file like object?"